apps/www/registry/default/ui/sidebar.tsx

```diff
@@ -198,7 +198,7 @@ const Sidebar = React.forwardRef<
           <SheetContent
             data-sidebar="sidebar"
             data-mobile="true"
-            className="w-[--sidebar-width] bg-sidebar p-0 text-sidebar-foreground [&>button]:hidden"
+            className={cn("w-[--sidebar-width] bg-sidebar p-0 text-sidebar-foreground [&>button]:hidden",className)}
```

Comment:
Make sure to change `apps/www/registry/new-york/ui/sidebar.tsx` as well

Then run the command to update the registry